### ISSUE TITLE
Fix nanoid vulnerability

### ISSUE THREAD

**Issue by markafitzgerald1** [markafitzgerald1/cribbage-trainer#225]
Current build output blocking any PR from getting merged:
```
╔═══════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════════╗
║                                                 === npm audit security report ===                                                 ║
║                                                                                                                                   ║
║ ID      │ Module │ Title                   │ Paths           │ Severity │ URL                                               │ Ex. ║
║ 1101092 │ nanoid │ Infinite loop in nanoid │ nanoid          │ low      │ https://github.com/advisories/GHSA-mwcw-c2x4-8c55 │ n   ║
╚═════════╧════════╧═════════════════════════╧═════════════════╧══════════╧═══════════════════════════════════════════════════╧═════╝
```
Let's fix this! Likely a simple `npm audit fix` will do the trick.

**Comment by markafitzgerald1** [comment]
Working to resolve this via #226.

**Comment by markafitzgerald1** [comment]
Resolved by #226.
